src/modules/media/elasticsearch/queryBuilder.ts

```diff
@@ -54,6 +82,28 @@ export class QueryBuilder {
 		};
 	}
 
+	/**
+	 * AND filter: https://stackoverflow.com/a/52206289/373207
+	 * @param elasticKey
+	 * @param readableKey
+	 * @param values
+	 */
+	private static generateAndFilter(
+		elasticKey: string,
+		readableKey: keyof SearchFilters,
+		value: string
+	): any {
```

Comment:
Deze mss iets stricter typen?

Reply:
Lijkt me beetje overkill in dit geval; gaat specifiek over elastic search query language, en kan nog wijzigen in de toekomst. Wordt ook niet elders in de code gebruikt dus minder zinvol.